modules/standard/Math.chpl

```diff
@@ -1290,6 +1290,22 @@ module Math {
     extern proc yn(n: c_int, x: real(64)): real(64);
     return yn(n.safeCast(c_int), x);
   }
+  
```

Comment:
Super nit -- Can you remove the extra trailing whitespace (there's 2 extra spaces here, and 1 in "It detects " in both comments below.)